tools/ci_connector_ops/ci_connector_ops/pipelines/builds/common.py

```diff
@@ -39,7 +39,10 @@ def image_name(self) -> Tuple:
     async def _run(self) -> StepResult:
         _, exported_tarball_path = await export_container_to_tarball(self.context, self.container)
         client = docker.from_env()
-        with open(exported_tarball_path, "rb") as tarball_content:
-            new_image = client.images.load(tarball_content.read())[0]
-        new_image.tag(self.image_name, tag=self.IMAGE_TAG)
-        return StepResult(self, StepStatus.SUCCESS)
+        try:
+            with open(exported_tarball_path, "rb") as tarball_content:
+                new_image = client.images.load(tarball_content.read())[0]
+            new_image.tag(self.image_name, tag=self.IMAGE_TAG)
+            return StepResult(self, StepStatus.SUCCESS)
+        except ConnectionError:
```

Comment:
Not related small fix: when building a lot of images locally in parallel the connection to docker host was unstable. I'd like to handle the connection error for a more explicit failure reason of this step.